### DISCUSSION TITLE
Failed to create super admin user

### DISCUSSION THREAD

**Discussion by AbinKumarPanda** [Budibase/budibase#1858] [Help] [answered]
Hi, I have deployed budibase on my local machine with docker but after entering credentials to create a super admin, the error pops up as super admin can't be created.

Please help.

**Comment by Rory-Powell** [comment]
Hi @AbinKumarPanda. Would you be able to include a screenshot of the error you are seeing?

If that doesn't help we might need to look at javascript console output or the application logs for more details

**Reply by AbinKumarPanda** [reply]
here I have attached the screenshot. Am I missing something ?

**Comment by AbinKumarPanda** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/38259762/123554560-33469380-d79e-11eb-8601-b121eae3438b.png)

**Reply by shogunpurple** [reply]
@AbinKumarPanda your URL is incorrect. You should be accessing your Budibase installation through the proxy:

localhost:10000


The problem you are having is because your setup is unable to talk to the service that creates users, since you are accessing it on 4002

**Reply by AbinKumarPanda** [reply]
Thanks a ton, got it working.
